grid2op/multi_agent/multiAgentEnv.py

```diff
@@ -152,6 +152,16 @@ def __init__(self,
                 self.agents, [None for _ in range(self.num_agents)]
             )
         )
+        self.done = dict(
+            zip(
+                self.agents, [False for _ in range(self.num_agents)]
+            )
+        )
+        self.info = dict(
+            zip(
+                self.agents, [{} for _ in range(self.num_agents)]
+            )
+        )
```

Comment:
nothing to do here...

Do you at least pay attention to what we said and what you are doing ?

Reply:
Done is not used but info is used in illegal action handling